src/cryptoadvance/specterext/notifications/service.py

```diff
@@ -0,0 +1,95 @@
+import logging
+
+from cryptoadvance.specter.services.service import (
+    Service,
+    devstatus_alpha,
+    devstatus_prod,
+    devstatus_beta,
+    ServiceRequisite,
+)
+
+# A SpecterError can be raised and will be shown to the user as a red banner
+from cryptoadvance.specter.specter_error import SpecterError
+from flask import render_template
+from cryptoadvance.specter.wallet import Wallet
+from flask_apscheduler import APScheduler
+from .notification_manager import NotificationManager
+from flask_login import current_user, AnonymousUserMixin
+
+logger = logging.getLogger(__name__)
+
+
+class NotificationsService(Service):
+    id = "notifications"
+    name = "Notifications Service"
+    icon = "notifications/img/notification.png"
+    logo = "notifications/img/notification.png"
+    desc = "Where a notifications grows bigger."
+    has_blueprint = True
+    blueprint_module = "cryptoadvance.specterext.notifications.controller"
+    devices = ["cryptoadvance.specterext.notifications.devices."]
```

Comment:
devices doesn't make sense here? Not sure where this is coming from.

Reply:
that was generated by `python3 -m cryptoadvance.specter ext ` ...